iaso/models/entity.py

```diff
@@ -57,6 +67,66 @@ def as_dict(self):
         }
 
 
+class InvalidLimitDateError(ValidationError):
+    pass
+
+
+class InvalidJsonContentError(ValidationError):
+    pass
```

Comment:
They should probably be in their own file and not here

Reply:
ok

Reply:
Well not really after thinking about it, and we didn't put errors in separate files in the project, but close to where they are triggered ...